File: python/62 PrimeFor.py
num=int(input("Enter a number: "))
isPrime=None
for i in range(2,num):
    if num%i==0:
        isPrime=False
        break
    else:
        isPrime=True
if isPrime:
    print("Prime number")
else:
    print("Not a prime number")

File: java/62 PrimeFor.java
package code;
import java.util.*;
public class Test {

	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter a number: ");
		int num = sc.nextInt();
		boolean isPrime=false;
		for(int i=2;i<num;i++) {
			if(num%i==0) {
				isPrime=false;
				break;
			}
			else {
				isPrime=true;
			}
		}
		if(isPrime) {
			System.out.println("Prime number");
		}
		else {
			System.out.println("Not a prime number");
		}
	}
}
